### DISCUSSION TITLE
Feature Request: search filter for PRs checks:passing checks:failing

### DISCUSSION THREAD

**Discussion by xdumaine** [community/community#34200] [Pull Requests]
Sometimes I have a lot of PRs and I'd love to be able to filter them by the status of their checks. It seems like it's technically feasible since PR results have an indicator for passing/failing/pending but I'd love to be able to search/filter by that status. Specifically talking about the ❌ and ✅ here:

![image](https://user-images.githubusercontent.com/833911/192283643-71db9b6c-cff5-42d9-8b93-2284c773720c.png)

**Comment by corneliusroemer** [comment]
This is actually possible! See https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-commit-status

![image](https://user-images.githubusercontent.com/25161793/224998763-7830df68-f6bb-443f-949f-26df2643c7ab.png)
